src/test/java/io/cryostat/net/web/http/api/v1/TargetsGetHandlerTest.java

```diff
@@ -131,5 +135,6 @@ public JMXServiceURL answer(InvocationOnMock args) throws Throwable {
                 gson.fromJson(
                         responseCaptor.getValue(), new TypeToken<List<ServiceRef>>() {}.getType());
         MatcherAssert.assertThat(result, Matchers.equalTo(targets));
+        Mockito.verify(resp).putHeader(HttpHeaders.CONTENT_TYPE, "application/json");
```

Comment:
I noticed only some tests call `verify(resp).putHeader()`. When is it appropriate to `verify()` that `putHeader()` was called?

Reply:
For V1 handlers, the behaviour is split between two places - for non-default values it's done by the specific implementation, otherwise it's done by the parent `AbstractAuthenticatedRequestHandler` in a different method. Since V1 tests generally directly invoke the `handleAuthenticated` method the default behaviour provided by the abstract parent class is not executed, so the test doesn't check for it either.

I should add a test that the parent class does do this default behaviour, though.